.github/workflows/bench.yml

```diff
@@ -22,4 +22,4 @@ jobs:
       run: curl -sfL https://raw.githubusercontent.com/knqyf263/cob/master/install.sh | sudo sh -s -- -b /usr/local/bin
 
     - name: Run Benchmark
-      run: cob --threshold 0.7 --base origin/master --bench-cmd make --bench-args test-performance
+      run: cob --threshold 0.7 --base origin/master --bench-cmd make --bench-args test-performance || exit 0
```

Comment:
This is temporary right? You will remove before the PR is merged?

Reply:
Yes, it's temporary, but I will remove it in the next PR because the test in this PR must fail. It is obvious and intended, but it looks red 🔴. So, I suppress the error once.
![image](https://user-images.githubusercontent.com/2253692/78353559-ebdd5180-75b2-11ea-8a5e-58be8f83fc32.png)